resources/crds/ripsaw_v1alpha1_uperf_cr.yaml

```diff
@@ -12,7 +12,7 @@ spec:
       pin: false
       pin_server: "node-0"
       pin_client: "node-1"
-      rerun: 1
+      samples: 1
       pair: 1
       test_types:
         - stream
```

Comment:
The spec.user and spec.elasticsearch sections are in the docs above, but they are not included in the example CR here.